Client/Frontend/Home/PanelDataObservers.swift

```diff
@@ -34,7 +36,7 @@ class ActivityStreamDataObserver: DataObserver {
     let profile: Profile
     weak var delegate: DataObserverDelegate?
     private var invalidationTime = OneMinuteInMilliseconds * 15
-    private var lastInvalidation = Date.now()
+    private var lastInvalidation: UInt64 = 0
```

Comment:
a blank date for lastInvalidation will force a refresh the first time its loaded (at launch)